### ISSUE TITLE
Automagic loading not working

### ISSUE THREAD

**Issue by EikeLueders** [MurhafSousli/ng2-progressbar#26]
The automagically bar is not working properly. I'm doing as shown below.
I also added `<ng-progress></ng-progress>` into my template. When I start the loading bar manually as described in the docs it's fine.

For my http requests I use the [angular http class](https://angular.io/docs/ts/latest/api/http/index/Http-class.html) and I'm on angular release 4.0.2.

```
import {HttpModule, BaseRequestOptions, BrowserXhr} from '@angular/http';
import {NgProgressModule, NgProgressCustomBrowserXhr} from "ng2-progressbar";

@NgModule({
    declarations: [
        AppComponent
    ],
    imports: [
        //...
        NgProgressModule
    ],
    providers: [
        {
            provide: BrowserXhr,
            useClass: NgProgressCustomBrowserXhr
        }
        //...
    ],
    bootstrap: [AppComponent]
})
```

Any ideas? Thanks. :)

**Comment by MurhafSousli** [comment]
Hi @EikeLueders, try to remove`HttpModule` from your imports

**Comment by EikeLueders** [comment]
Hi, thanks for your quick response 👍 
I tried your suggestions, but it led to an provider error. Actually, the `HttpModule` is required.
However, I was able to solve my issue. I had activated my `MockBackend` for testing. With this the loader doesn't work. I assume because there are no real http requests. Once, I deactivated it and request data from the real API all is fine.

Thanks again for your great work on the progress bar!
